modules/gdscript/gdscript_formatter.cpp

```diff
@@ -0,0 +1,231 @@
+/*************************************************************************/
+/*  gdscript_formatter.cpp                                               */
```

Comment:
This should likely go in `modules/gdscript/editor/` so that's in only compiled in the editor, and not export templates. The relevant code should also be enclosed in `#ifdef TOOLS_ENABLED` in `main.cpp` then.